templates/cloud/public-cloud-survey.html

```diff
@@ -0,0 +1,39 @@
+{% extends "cloud/_base-cloud.html" %}
+
+{% block title %}Public Cloud Operations Survey{% endblock %}
+
+{% block meta_description %}Public Cloud Operations Survey{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1tKpr0dpZx2K7S2wiBF7QKONNLs2fxz0sFoautszQR1o/edit#heading=h.fmnr02fzg6id{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped is-bordered">
+    <div class="row u-vertically-center">
+        <div class="col-6">
```

Comment:
I think the cols look a bit better like this to spread the text out a bit
```suggestion
        <div class="col-7">
```